OpenShift route configuration

G0ntschow opened the pull request:
Hi, we're using NeuVector on OpenShift. 
Currently it's not possible to manage routes for the web interface or for federation. 
Like discussed with customer management I created a Pull request to point out what we need in the Helm Chart form customer perspective.

becitsthere commented:
Thanks for the PR. A couple of suggestions,
1. Because controller.apisvc.type is used in non-route services, it is better to have a dedicate controller.apisvc.route.enabled variable.
2. Should be neuvector-svc-controller-fed-managed instead of neuvector-svc-controller-fed-master in line 67
3. Not all new variables are added to values.yaml. README.md need to be updated as well.
4. Bump up core chart version is required by repository actions, if anything of the chart is changed.

G0ntschow commented:
I updated the the Pull Request regarding your suggestions. I also added a route.enabled variable to the federation routes as well to keep the overall structure of the values file consistent.
